bluepymm/prepare_combos/parse_files.py

```diff
@@ -128,10 +128,10 @@ def read_mm_recipe_yaml(recipe_filename):
     mecombos = pandas.DataFrame(columns=["layer", "fullmtype", "etype"])
     for region in recipe['neurons']:
         for etype in region['traits']['etype'].keys():
-            n_combos = len(mecombos)
-            mecombos.loc[n_combos, 'layer'] = region['traits']['layer']
-            mecombos.loc[n_combos, 'fullmtype'] = region['traits']['mtype']
-            mecombos.loc[n_combos, 'etype'] = etype
+            n = len(mecombos)
```

Comment:
why use 'n'?

Reply:
to fit the 80 limit ;)

Reply:
I'd just put a newline before 'region' or so then :-)